frontend/Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM node:19-alpine
 
 # Set work dir for relative paths
-RUN mkdir -p /app
 WORKDIR /app
 
 # Install dependencies
-COPY ./frontend/package*.json /app
+COPY ./package*.json .
+COPY ./yarn.lock .
```

Comment:
Added the lock file for yarn so that it's also referenced when the dependencies are getting resolved in the image.